infrastructure/modules/workers_with_sidecar/variables.tf

```diff
@@ -64,22 +54,19 @@ variable "security_group_ids" {
   default = []
 }
 
-variable "app_image" {
-  type = string
-}
-
-variable "app_name" {
-  type = string
-}
-
-variable "app_cpu" {
-  type    = number
-  default = 512
-}
-
-variable "app_memory" {
-  type    = number
-  default = 1024
+variable "apps" {
```

Comment:
Will this change impact other consumers of this module within the catalogue repo?

Reply:
This is the only consumer of the module - consolidating the modules within the catalogue repo is a thread I've started pulling a few times only to find that it's really quite a big pain to do so. A project for a rainy day, perhaps.